python/geo2png.py

```diff
@@ -0,0 +1,121 @@
+# trafo -> png
+# Original source https://github.com/vrandezo/wikidata-analytics/blob/6896d639bd124eec7571f1179f27ae2b6bfaa435/geo2png.py
```

Comment:
All I have to go on is this
https://github.com/wmde/wikidata-analysis/blob/addshoreRewrite/html/map/index.html#L123
The original tool says that all code is PD